tests/resources/cfg.yml

```diff
@@ -78,6 +78,10 @@ line_item:
   name: "Prebid-{{ bidder_name }}-{{ media_type }}-{{ time }} @ {{ cpm }}"
   item_type: "price_priority"
   # Optional
+  # goal: # required if sponsorship item_type
+    # units: 5
+    # unitType: "VIEWABLE_IMPRESSIONS"
+    # goalType: "LIFETIME"
```

Comment:
This should be added here as well:
https://github.com/prebid/line-item-manager/blob/master/line_item_manager/conf.d/line_item_manager.yml#L103